NvAPIWrapper/Display/PathInfo.cs

```diff
@@ -21,7 +21,7 @@ public class PathInfo : IEquatable<PathInfo>
         /// <param name="resolution">Display resolution</param>
         /// <param name="colorFormat">Display color format</param>
         /// <param name="targetInfos">Target configuration informations</param>
-        public PathInfo(Resolution resolution, ColorFormat colorFormat, PathTargetInfo[] targetInfos)
+        public PathInfo(Resolution resolution, Format colorFormat, PathTargetInfo[] targetInfos)
```

Comment:
Changing `ColorFormat` to `Format` breaks backward compatibility and forces a new major version. Isn't this possible to change the new type to something like `HDRColorFormat`?

Reply:
> Hey, thanks for the PR. Are these methods public? Can you point me to the documentation? Quite complicated just for HDR.

Tell me about it! All this from trying to find a way to control the settings mentioned from the command line.

NvAPI_Disp_ColorControl
https://docs.nvidia.com/gameworks/content/gameworkslibrary/coresdk/nvapi/group__dispcontrol.html#gaea699b512c7a7c540e5c0dd6ae4bcd56

NvAPI_Disp_HdrColorControl
https://docs.nvidia.com/gameworks/content/gameworkslibrary/coresdk/nvapi/nvapi_8h.html#a6175ff4b4f58c1787d1964abf43ebb84

NvAPI_Disp_GetHdrCapabilities
https://docs.nvidia.com/gameworks/content/gameworkslibrary/coresdk/nvapi/nvapi_8h.html#af7f3f64f639482c9d79e6b18358353de